### PR TITLE
show picture-in-picture button while share screen 

### PR DESCRIPTION
**Pull request by iospro**
show picture-in-picture button while share screen to allow compact JitsiMeetView
(significant when SDK is embedded in some host app)

### CODE REVIEW

**Comment by saghul** [comment]
We don't want to allow that. I think an extra flag would be acceptable.

**Comment by iospro** [comment]
And what is the reason now to hide this while translation?

File: react/features/base/flags/constants.ts [hunk 1]
```diff
@@ -160,6 +160,12 @@ export const OVERFLOW_MENU_ENABLED = 'overflow-menu.enabled';
  */
 export const PIP_ENABLED = 'pip.enabled';
 
+/**
+ * Flag indicating if Picture-in-Picture button should be shown while translation.
```
Comment (saghul) [inline react/features/base/flags/constants.ts:164]: This description sounds very confusing. There is nothing related to translations here. What you want is to enable it while screen sharing.

File: react/features/base/flags/constants.ts [hunk 2]
```diff
@@ -160,6 +160,12 @@ export const OVERFLOW_MENU_ENABLED = 'overflow-menu.enabled';
  */
 export const PIP_ENABLED = 'pip.enabled';
 
+/**
+ * Flag indicating if Picture-in-Picture button should be shown while screen sharing.
+ * Default: disabled (false).
+ */
+export const PIP_AT_SCREEN_SHARING_ENABLED = 'pip-at-screen-sharing.enabled';
```
Comment (saghul) [inline react/features/base/flags/constants.ts:167]: Please call this PIP_WHILE_SCREEN_SHARING_ENABLED

**Review by saghul — CHANGES_REQUESTED** [review]

**Review by saghul — APPROVED** [review]

**Pull request merged**